wknml/__init__.py

```diff
@@ -324,28 +323,28 @@ def dump_parameters(xf, parameters):
         "z": str(parameters.scale[2]),
     })
 
-    if parameters.offset is not None:
+    if parameters.offset is not None and parameters.offset != (0, 0, 0):
```

Comment:
Are you sure that setting the offset to None is a good idea? When we take (0, 0, 0) as the default value it won't be saved to the disk and when we read the NML from the disk again then (0, 0, 0) will be set as a default value since the offset is missing in the file.